cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```

Comment:
Let's lose Cypress's boilerplate (if needed, their comment at the top already has links to relevant docs)
```suggestion
import './commands';
```

Reply:
Removed, and deleted _commands.js_

Reply:
No strong objections, but FYI something that's come up for me in the past when I was setting up Enterprise Search's Cypress proj was that every time I used `cypress open` Cypress kept autogenerating boilerplate files in my local (specifically the `fixtures` and I think the `support` folder, I can't remember if `commands.js` was part of that also): https://github.com/cypress-io/cypress/issues/501

Might be worth keeping an eye on and if it re-adding these files with minimal boilerplate if it does that still